src/common/IOSLauncher.ts

```diff
@@ -79,9 +80,19 @@ export class IOSLauncher {
                 return IOSUtils.waitUntilDeviceIsReady(deviceUDID);
             })
             .then(() => {
+                const address =
+                    appConfig && appConfig.preview_server_enabled === true
+                        ? 'http://localhost' // TODO: dynamically determine server address
+                        : undefined;
+
+                const port =
+                    appConfig && appConfig.preview_server_enabled === true
+                        ? serverPort
+                        : undefined;
+
                 if (PreviewUtils.isTargetingBrowser(targetApp)) {
                     const compPath = PreviewUtils.prefixRouteIfNeeded(compName);
-                    const url = `http://localhost:3333/lwc/preview/${compPath}`;
+                    const url = `${address}:${serverPort}/lwc/preview/${compPath}`;
```

Comment:
The assumption here is that `serverPort` will always have a value and not undefined, right? And that's the reason `port` is not used instead.

Reply:
yes that's correct. I also could have used `port` instead since if we're inside this block then `port = serverPort`

Reply:
But then the `port` could possibly be undefined. If so do we need another formatting string for `url` without the port number?

Reply:
There are 2 ways about it. Either keep it the way I have here, or define `port` like below and use it instead:
```
const port =
    PreviewUtils.isTargetingBrowser(targetApp) ||
    (appConfig && appConfig.preview_server_enabled === true)
        ? serverPort
        : undefined;
```

Reply:
If we can avoid the scenario where the port is undefined and won't be used for formatting string I am good either way.